pkg/sdn/api/v1/conversion.go

```diff
@@ -8,6 +8,34 @@ import (
 )
 
 func addConversionFuncs(scheme *runtime.Scheme) error {
+	if err := scheme.AddFieldLabelConversionFunc("network.openshift.io/v1", "ClusterNetwork",
```

Comment:
we want to make the new api group version of this compatible with the upstream networking?